code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm

```diff
@@ -0,0 +1,68 @@
+#define STATUS_EFFECT_TRICKED /datum/status_effect/tricked
+/mob/living/simple_animal/hostile/abnormality/laetitia
+	name = "Laetitia"
+	desc = "A wee witch."
+	icon = 'ModularTegustation/Teguicons/tegumobs.dmi'
+	icon_state = "laetitia"
+	maxHealth = 600
+	health = 600
+	threat_level = HE_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(40, 45, 50, 50, 50),
+		ABNORMALITY_WORK_INSIGHT = 40,
+		ABNORMALITY_WORK_ATTACHMENT = list(60, 60, 60, 65, 65),
+		ABNORMALITY_WORK_REPRESSION = 0
+			)
+	work_damage_amount = 8
+	work_damage_type = WHITE_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/prank,
+		/datum/ego_datum/armor/prank
+		)
+
+/mob/living/simple_animal/hostile/abnormality/laetitia/neutral_effect(mob/living/carbon/human/user, work_type, pe)
```

Comment:
Know what would be funny? If it was a random chance, because right now neutral result is a guarantee for this to happen, so people would either hide in a super secluded area or gather entire team to quickly kill it when it appears.
With it being a random chance(even if it's 75-90%) it could solve this, making it so you can never be too sure.

Reply:
It's done